s3-expand

```diff
@@ -69,6 +69,7 @@ cmd_check_require MKDIR mkdir
 cmd_check_require TAR tar
 cmd_check_require RM rm
 cmd_check_require MKTEMP mktemp
+S3CFG_FILE=/tmp/.s3cfg
```

Comment:
Could also use `mktemp` to create a temporary file in the system temporary directory. Your solution works because it's running in a clean file system.